test/automation/src/sql/configurePythonDialog.ts

```diff
@@ -23,6 +23,9 @@ export class ConfigurePythonDialog extends Dialog {
 	}
 
 	async waitForPageOneLoaded(): Promise<void> {
+		const pageOne = 'div.wizardNavigation-container a[title="Configure Python Runtime"]';
+		await this.code.waitAndClick(pageOne);
```

Comment:
This fixes the issue?

I'm not sure I like having to do extra stuff like this just to get around what appears to be a problem with the tests. Smoke tests should be mimicing what users are doing - but this is adding a step that most users would never actually do themselves and so may potentially hide (or expose other issues)

(having tests that click through the navigation buttons to verify they're working correctly isn't a bad idea, but that's outside the scope of this PR).

It would be best to actually try and figure out what the issue was and fix it - especially since it could very well be something that breaks other tests as well.

Reply:
I ran an ad hoc build and the Python test passed but this issue only happens sometimes so it is hard to say for sure that this extra step will fix the issue.

I agree that adding this extra step is hiding the underlying issue. Right now I am not sure if the issue is in the Configure Python Wizard code or in the test itself. Based on the test logs, there are no steps that could be moving the wizard to the next page so I will look more into the way the wizard is being initialized to see if there might be something weird going on there.

Reply:
I'll close this for now and open up a separate PR for the screenshots fix.

Reply:
Sounds good, I think our first step here is to make sure we have logging available to determine what the issue is (if that's possible to add). So that's where I'd suggest we look at making improvements until we know what's actually happening.